README.adoc

```diff
@@ -358,9 +358,9 @@ You will receive a response that contains the JSON object of the updated crew me
 
 === Implementing the delete operation
 
-The [hotspot=remove]`remove()` method handles the implementation of the `delete` operation. After the `Crew` collection is retrieved, a [hotspot=queryDelete]`document` is created with the specified object id to query the collection. Because the object id is unique in the collection, only one document is deleted. After the document is deleted, the [hotspot=deleteOne]`MongoCollection.deleteOne()` method returns a `DeleteResult`, which determines how many documents were deleted. If the deleted count is zero, then the object id doesn't exist.
+The [hotspot=remove]`remove()` method handles the implementation of the delete operation. After the `Crew` collection is retrieved, a [hotspot=queryDelete]`document` is created with the specified object id to query the collection. Because the object id is unique in the collection, only one document is deleted. After the document is deleted, the [hotspot=deleteOne]`MongoCollection.deleteOne()` method returns a `DeleteResult`, which determines how many documents were deleted. If the deleted count is zero, then the object id doesn't exist.
```

Comment:
Break into multiple lines. 

Also not sure if we should add backticks to `id` as its a property.